### PR TITLE
User story 46

### PR DESCRIPTION
**Pull request by muydanny**
Random syntax error

ALMOST DONE FOR THE LOVE OF GOD

Damp Hollows

### CODE REVIEW

**Review by Gallup93 — APPROVED** [review]
LAST USER STORY

File: spec/features/merchant/items/index_spec.rb [hunk 1]
```diff
@@ -173,7 +173,55 @@
...
+      expect(current_path).to eq("/merchant/items/new")
+      expect(page).to have_content("Inventory must be greater than -1")
+    end
   end
 end
 
 
+
+
+# User Story 46, Merchant cannot add an item if details are bad/missing
```
[the remark this answers is not in the document]
Comment (stellakunzang) [reply spec/features/merchant/items/index_spec.rb:220]: yep, looking forward to deleting these comments for sure

**Review by stellakunzang — APPROVED** [review]

**Pull request merged**